server/views/pages/prisons/addSessionTemplate.njk

```diff
@@ -0,0 +1,236 @@
+{% extends "../../partials/layout.njk" %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{%- from "govuk/components/back-link/macro.njk" import govukBackLink -%}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/date-input/macro.njk" import govukDateInput %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+
+
+{% set pageHeaderTitle = 'Add session template' %}
+{% set pageTitle = applicationName + " - " + pageHeaderTitle %}
+
+{% set primaryNavItems = [
+  {
+    text: "Home",
+    href: "/"
+  },
+  {
+    text: "Prisons",
+    href: "/prisons",
+    active: true
+  }
+  ] %}
+{% set backLinkHref = "/prisons/" + prisonId + "/session-templates" %}
+
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <h1 class="govuk-heading-l">
+        <span class="govuk-caption-l">{{ prisonName }}</span>
+        {{ pageHeaderTitle }}
+      </h1>
+
+      <form action="/prisons/{{prisonId}}/session-templates/add" method="POST" novalidate>
+        <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+
+        {{ govukInput({
+          label: {
+              text: "Template name",
+              classes: "govuk-label--m"
+          },
+          hint: {
+              text: "What is the name of the template?"
```

Comment:
Might want this field slightly wider. It'll be 100 chars I believe. Perhaps don't need the hint text for this one (reduce clutter?) 🤷‍♂️